RNiTunes/RNiTunes.m

```diff
@@ -166,6 +170,8 @@ @implementation RNiTunes
         NSString *genre = [song valueForProperty: MPMediaItemPropertyGenre]; // filterable
         NSString *duration = [song valueForProperty: MPMediaItemPropertyPlaybackDuration];
         NSString *playCount = [song valueForProperty: MPMediaItemPropertyPlayCount];
+        NSURL *url = [song valueForProperty: MPMediaItemPropertyAssetURL];
+        NSString *assetUrl = url.absoluteString;
```

Comment:
What's the value building up all these properties on this one object and then pretty shortly dereferencing the same properties to construct these different strings?

Reply:
Not sure, maybe just so we can reference the vars and put them up here?